<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>juego</class>
 <widget class="QMainWindow" name="juego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PIG GAME</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Player_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 11pt &quot;Ubuntu Mono&quot;;</string>
    </property>
    <property name="title">
     <string>Jugador 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Flautista">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="imagenes.qrc">:/Imagenes/CerditoFlautista.webp</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="Player_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 11pt &quot;Ubuntu Mono&quot;;</string>
    </property>
    <property name="title">
     <string>Jugador 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="Violinista">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="imagenes.qrc">:/Imagenes/CerditoViolinista.webp</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="Score_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>SCORE 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcd_score1"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="Score_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>SCORE 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcd_score2"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="Dado">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Dado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcd_dado"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="bt_reiniciar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reiniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_lanzar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Lanzar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_pasar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pasar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Tiros">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Tiros</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcd_tiros"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>bt_lanzar</tabstop>
  <tabstop>bt_pasar</tabstop>
  <tabstop>bt_reiniciar</tabstop>
 </tabstops>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
